ext/intl/tests/dateformat_get_best_pattern.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+IntlDateFormatter::getBestPattern()
+--SKIPIF--
+<?php
+if (!extension_loaded('intl'))	die('skip intl extension not enabled'); ?>
+<?php if (version_compare(INTL_ICU_VERSION, '50.1.2', '<')) die('skip for ICU < 50.1.2'); ?>
+--FILE--
+<?php
+
+ini_set("intl.error_level", E_WARNING);
+ini_set("intl.default_locale", "en_US");
+
+echo IntlDateFormatter::getBestPattern("jjmm"), "\n";
+echo IntlDateFormatter::getBestPattern("jjmm", "de_DE"), "\n";
+echo IntlDateFormatter::getBestPattern("YYYYMMMdd"), "\n";
+echo IntlDateFormatter::getBestPattern("YYYYMMMdd", "de_DE"), "\n";
+echo datefmt_get_best_pattern("YYYYMMMdd"), "\n";
+
+echo IntlDateFormatter::getBestPattern(""), "\n";
+
+echo IntlDateFormatter::getBestPattern(), "\n";
```

Comment:
Catching exceptions and then echoing the message is preferred. And then you can even get rid of the `EXPECTF` in favor of `EXPECT`.